gradle/libs.versions.toml

```diff
@@ -1,21 +1,21 @@
 [versions]
 assertj = "3.25.3"
-aws = "2.20.18" # this is in mapping with iceberg repo.
+aws = "2.24.5" # this is in mapping with iceberg repo.
 checkstyle = "10.14.2"
 errorprone = "2.26.1"
 errorproneSlf4j = "0.1.23"
 googleJavaFormat = "1.21.0"
 guava = "32.1.3-jre"
 hadoop = "2.7.3" # this is in mapping with iceberg repo.
 hive = "2.3.9" # this is in mapping with iceberg repo.
-iceberg = "1.3.1"
+iceberg = "1.5.0"
 immutables = "2.10.1"
 jacoco = "0.8.11"
 jandex = "3.1.7"
 junit = "5.10.2"
 logback = "1.5.3"
 logcaptor = "2.9.2"
-nessie = "0.60.1"
+nessie = "0.79.0"
```

Comment:
Using 0.79.0 instead of 0.77.1 because it has removed deprecated view classes (so the jar is lighter) and still compatible with Iceberg 1.5.0

Reply:
sgtm

generally wondering about compatibility though:

does the catalog-migrator now only support targeting iceberg 1.5.0 ?
if yes do we need to create/update some docs?
if no, do we have backward compatibility tests?